libs/designer/src/lib/ui/configuretemplate/review/TemplateReviewList.tsx

```diff
@@ -73,16 +73,16 @@ export const TemplateReviewList = () => {
   const { connectorKinds, stateTypes, resourceStrings: templateResourceStrings } = useTemplatesStrings();
   const resources = { ...templateResourceStrings, ...connectorKinds, ...stateTypes, ...useResourceStrings(), ...intlText };
 
-  const statusAndPlanItems: TemplatesSectionItem[] = useStatusAndPlanItems(resources);
+  const BasicsItems: TemplatesSectionItem[] = useBascisItems(resources);
```

Comment:
```suggestion
  const BasicsItems: TemplatesSectionItem[] = useBasicsItems(resources);
```

nit: This one may need to be fixed in other places as well